### ISSUE TITLE
[Bug]: Empty home page on release 0.1.1

### ISSUE THREAD

**Issue by SimonMarquis** [android/nowinandroid#863]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is there a StackOverflow question about this issue?

- [X] I have searched StackOverflow

### What happened?

A bug happened!

![image](https://github.com/android/nowinandroid/assets/1921278/1e40dbc1-843d-405d-9029-20eba1690edc)

### Relevant logcat output

```shell
Failed to evaluate a suspendRunCatchingBlock. Returning failure Result
java.lang.ClassCastException: java.lang.Class cannot be cast to java.lang.reflect.ParameterizedType
	at ad.t.b(SourceFile:2486)
	at ad.y0.b(Unknown Source:30)
	at ad.x0.invoke(SourceFile:45)
	at java.lang.reflect.Proxy.invoke(Proxy.java:1006)
	at com.google.samples.apps.nowinandroid.core.network.retrofit.$Proxy3.b(Unknown Source)
	at com.google.samples.apps.nowinandroid.core.network.retrofit.f.a(Unknown Source:2)
	at o9.n.i(SourceFile:47)
	at o9.n.w(SourceFile:21)
	at m9.b.i(SourceFile:117)
	at m9.b.x(SourceFile:24)
	at k4.h0.o0(Unknown Source:52)
	at k4.h0.D(SourceFile:75)
	at o9.q.a(Unknown Source:30)
	at pa.d.i(SourceFile:33)
	at ib.a.h(Unknown Source:8)
	at kotlinx.coroutines.j0.run(Unknown Source:104)
	at kotlinx.coroutines.internal.f.run(Unknown Source:11)
	at kotlinx.coroutines.scheduling.i.run(Unknown Source:2)
	at kotlinx.coroutines.scheduling.a.run(SourceFile:77)
Failed to evaluate a suspendRunCatchingBlock. Returning failure Result
java.lang.ClassCastException: java.lang.Class cannot be cast to java.lang.reflect.ParameterizedType
	at ad.t.b(SourceFile:2486)
	at ad.y0.b(Unknown Source:30)
	at ad.x0.invoke(SourceFile:45)
	at java.lang.reflect.Proxy.invoke(Proxy.java:1006)
	at com.google.samples.apps.nowinandroid.core.network.retrofit.$Proxy3.c(Unknown Source)
	at com.google.samples.apps.nowinandroid.core.network.retrofit.f.c(Unknown Source:2)
	at o9.u.i(SourceFile:38)
	at o9.u.w(SourceFile:21)
	at m9.b.i(SourceFile:117)
	at m9.b.x(SourceFile:24)
	at k4.h0.o0(Unknown Source:52)
	at k4.h0.D(SourceFile:75)
	at o9.w.a(Unknown Source:25)
	at pa.c.i(SourceFile:33)
	at ib.a.h(Unknown Source:8)
	at kotlinx.coroutines.j0.run(Unknown Source:104)
	at kotlinx.coroutines.internal.f.run(Unknown Source:11)
	at kotlinx.coroutines.scheduling.i.run(Unknown Source:2)
	at kotlinx.coroutines.scheduling.a.run(SourceFile:77)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dturner** [comment]
Fixed in #861

**Comment by SimonMarquis** [comment]
Confirmed to be fixed in release 0.1.2 👍
